### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by fabaff**
A couple of updates for the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,62 +2,61 @@
...
 
-There is a "tests" directory containing some unit tests for some of the classes and functions in this application.
+There is a `tests` directory containing some unit tests for some of the classes and functions in this application.
 
-Additionally, the application has been tested on the following systems and python versions:
+Additionally, the application has been tested on the following systems and Python versions:
 
-* Ubuntu 16.04 (python 3.5.2, python 2.7.12)
+* Ubuntu 16.04 (Python 3.5.2, Python 2.7.12)
+* Fedora 27 (Python 3.6.3)
```
Comment (skymaya) [inline README.md]: Will approve this change.

File: README.md [hunk 2]
```diff
@@ -2,62 +2,61 @@
 
 ## Synopsis
 
-A command line tool and import library for performing unit conversions.
+A command line tool and a library for performing unit conversions.
 
 ## Testing
 
-There is a "tests" directory containing some unit tests for some of the classes and functions in this application.
+There is a `tests` directory containing some unit tests for some of the classes and functions in this application.
```
Comment (skymaya) [inline README.md]: Will approve this change.

File: README.md [hunk 3]
```diff
@@ -2,62 +2,61 @@
...
 
-Additionally, the application has been tested on the following systems and python versions:
+Additionally, the application has been tested on the following systems and Python versions:
```
Comment (skymaya) [inline README.md]: Will approve this change.

File: README.md [hunk 4]
```diff
@@ -2,62 +2,61 @@
...
+* Fedora 27 (Python 3.6.3)
 
 ## Known Issues
 
 * Fully untested on Windows, Mac, and BSD at this time
 
 ## Installation
 
-Install with pip:
+Install with `pip`:
```
Comment (skymaya) [inline README.md]: Will approve this change.

File: README.md [hunk 5]
```diff
@@ -2,62 +2,61 @@
...
+```
 
 ## Usage
 
-After installation is completed, you may run the unitconvert tool in a terminal using the following command:
-
-`unitconvert -a AMOUNT -f UNIT_FROM -t UNIT_TO`
-
-To print version information:
+After installation is completed, you may run the `unitconvert` tool in a terminal using the following command:
```
Comment (skymaya) [inline README.md]: Will approve this change.

File: README.md [hunk 6]
```diff
@@ -2,62 +2,61 @@
...
 
-`unitconvert -h`
+* -a AMOUNT: Amount to convert
+* -f UNIT_FROM: Unit to convert from
+* -t UNIT_TO: Unit to convert to
+* -v VERSION: Optional, print version info
+* -h with no other arguments: print help info
 
-The unit libraries may also be imported and used in Python scripts to perform conversions. Example:
+The `unitconvert` libraries may also be imported and used in Python scripts to perform conversions. Example:
```
Comment (skymaya) [inline README.md]: Will approve this change.

**Comment by fabaff** [comment]
Sorry, I don't understand what you mean by "Will approve this change."?

**Pull request closed without merging**